### ISSUE TITLE
Running game on Ubuntu 11.04 results in menu popping up whatever you do.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sunfall/endgame-singularity#120]
```
What steps will reproduce the problem?
1. Run the Ubuntu 11.04 version from the repo.

2. Try starting a new game other then normal; all locations are already there 
at the start, and the game simply keeps giving you the menu regardless of what 
you do

3. After a while, playing on normal, opening up 
bases/research/finance/knowledge causes the menu to pop up, instead of the 
intended function.

The game is practically non-functional for me. Other Ubuntu users have reported 
the same problem.


```

Original issue reported on code.google.com by `danteash...@gmail.com` on 13 May 2011 at 10:48

**Comment by GoogleCodeExporter** [comment]
```
Having this same issue,on Ubuntu after having played on normal for a while the 
menu starts appearing anywhere you click.
```

Original comment by `sabrek...@gmail.com` on 10 Aug 2011 at 1:54

**Comment by GoogleCodeExporter** [comment]
```
Ditto here.

Output from console attached.
```

Original comment by `B.Call...@gmail.com` on 18 Aug 2011 at 7:57

Attachments:
* [traceback.txt](https://storage.googleapis.com/google-code-attachments/endgame-singularity/issue-120/comment-2/traceback.txt)

**Comment by GoogleCodeExporter** [comment]
```
A patch for this issue from 
https://bugs.launchpad.net/ubuntu/+source/singularity/+bug/576504 attached:
```

Original comment by `non...@gmail.com` on 24 Sep 2011 at 10:33

Attachments:
* [sing_p_not_vs_src.patch](https://storage.googleapis.com/google-code-attachments/endgame-singularity/issue-120/comment-3/sing_p_not_vs_src.patch)

**Comment by GoogleCodeExporter** [comment]
```
This was fixed in 0.30c.
```

Original comment by `phil.bor...@gmail.com` on 29 Mar 2012 at 7:04
* Changed state: **Fixed**
